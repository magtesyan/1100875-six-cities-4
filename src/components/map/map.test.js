import renderer from "react-test-renderer";
import React from "react";

import Map from "./map.jsx";

const offers = [
  {
    coordinates: [52.3909553943508, 4.85309666406198],
  },
];
const city = [52.38333, 4.9];

const createMapBlock = () => {
  const div = global.document.createElement(`div`);
  div.setAttribute(`id`, `map`);
  global.document.body.appendChild(div);
};

it(`Map is rendered correctly`, () => {
  createMapBlock();
  const tree = renderer
    .create(
        <Map
          city={city}
          offers = {offers}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
