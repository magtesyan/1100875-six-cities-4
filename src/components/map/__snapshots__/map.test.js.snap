// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map is rendered correctly 1`] = `
<section
  className="cities__map map"
>
  <div
    id="map"
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  />
</section>
`;
